js/src/abi/spec/param.js

```diff
@@ -31,6 +31,12 @@ export default class Param {
   }
 
   static toParams (params) {
-    return params.map((param) => new Param(param.name, param.type));
+    return params.map((param) => {
```

Comment:
Could we just add a relevant test that verifies both behaviours? (these should be one that just verifies the original)